lib/cbx_loco/loco_adapter.rb

```diff
@@ -0,0 +1,218 @@
+require 'rest-client'
+require 'time'
+require 'json'
+require 'yaml'
+require 'colorize'
+require 'byebug'
```

Comment:
Unneeded require, as byebug is not used in the code. I understand that it was used during development, but the require should not make it to "production" code.